src/Card/Card.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <div>
+    <div class="card" :style="cardStyle">
+      <div :class="['image', `image__side__${imgSide}`]">
+        <img class="image__img" v-if="imgSrc" :src="imgSrc" alt="">
+      </div>
+      <div class="text">
+        <div v-if="overline" class="overline">
+          {{overline}}
+        </div>
+        <div v-if="title" class="title">
+          {{title}}
+        </div>
+        <div :class="['body', `body__canexpand__${!!(canExpand)}`, `body__expanded__${!!(isExpanded)}`]" ref="body" @transitionend="transitionend">
+          <!-- @slot Main content of the card -->
+          <slot/>
+          <transition name="fade">
+            <div class="body__more" @click="expand" v-if="canExpand && !isExpanded">read more</div>
```

Comment:
![image](https://user-images.githubusercontent.com/1021101/75818422-b217f180-5d4d-11ea-9178-5685782838f1.png)

Not sure how we display `read more` is the conventional way 🤔 @fadeev

Reply:
Well, we're not using this component and probably won't be it at all, so it doesn't matter.